apps/lib/apps.c

```diff
@@ -581,31 +581,35 @@ EVP_PKEY *load_key(const char *uri, int format, int may_stdin,
     if (desc == NULL)
         desc = "private key";
 
-    if (format == FORMAT_ENGINE) {
+    if (format == FORMAT_ENGINE)
         uri = allocated_uri = make_engine_uri(e, uri, desc);
-    }
```

Comment:
No need to make this change.

Reply:
This improves adherence to the coding style and 
makes `check-format.pl apps/lib/apps.c` happy :smile: